R/half.life.R

```diff
@@ -32,6 +32,12 @@
 #' @param tlast Time of last concentration above the limit of
 #'   quantification (will be calculated and included in the return data
 #'   frame if not given)
+#' @param time.dose Only use points with time > time.dose + duration.dose
+#'   for half-life calculation. This can be used to exclude points from
+#'   the calculation that are affected by the dosing regimen.
+#' @param duration.dose The duration of the dose administration. Points
+#'   within this time frame after time.dose will be excluded from the
+#'   half-life calculation.

```

Comment:
If possible, please use `# @inheritParams` with a source function that defines the parameters. That ensures consistency of documentation. (I don't specifically know that these are defined elsewhere.)